MAVProxy/modules/lib/mp_menu.py

```diff
@@ -330,6 +330,16 @@ def call(self):
             dlg = wx.FileDialog(None, self.title, '', "", self.wildcard, flagsMapped[0])
         else:
             dlg = wx.FileDialog(None, self.title, '', "", self.wildcard, flagsMapped[0]|flagsMapped[1])
+class MPMenuCallDirDialog(object):
```

Comment:
please have at least one blank line from the previous class

Reply:
Hi @tridge,

Thanks for that, fixed!